app/page/webauthn/devtool.tsx

```diff
@@ -187,7 +187,20 @@ export function WebAuthnDevTool() {
         <div></div>
       ) : (
         <div>
-          <p>WebAuthn credential: {json.stringify(credential, null, 2)}</p>
+          <p>
+            WebAuthn credential:{" "}
+            {json.stringify(
+              credential,
+              (key, value) => {
+                // Filtering out properties
+                if (value instanceof Uint8Array) {
+                  return Buffer.from(value).toString("hex")
+                }
+                return value
+              },
+              2
+            )}
```

Comment:
nit: I saw many places use stringify to log info. It may be worth to extract this as function and apply it to all the places.

Reply:
I think this could be a separate PR.
Would it be appropriate to add a function named stringify2() in #80 PR?

Reply:
The core point is about the conversion of Uint8Array to hex, but not the 2 spaces indent. If you want to provide a single standard in formatting json, I would suggest to name it as `format`, and also include the Uint8Array as intrinsic replacer.

Reply:
Done! Updated PR!